frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -11,48 +10,132 @@ interface TopNavBarProps {
     className?: string
 }
 
+const menuToggleStyles = {
+    width: 150,
+    height: 50,
+    opacity: 0.7,
+    cursor: 'pointer',
+    whiteSpace: 'nowrap' as const,
+    userSelect: 'none' as const,
+};
+
+const Wrapper = styled.div({
+    position: 'relative',
+    display: 'inline-block',
+});
+
 export const StyledLink = styled(NavLink)({
     textDecoration: 'none',
-    color: colors.text,
+    color: 'white',
+    fontFamily: 'System-ui',
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    textAlign: 'left',
+});
+
+const NavbarStyledLink = styled(NavLink)({
+    textDecoration: 'none',
+    color: 'white',
+    fontFamily: 'System-ui',
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    textAlign: 'left',
+    position: 'relative',
     '&.active': {
-        color: colors.blue,
+        color: 'white',
     },
-})
-
-const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'))
-const AccountLinks = loadAsync('Account Links', () => import('./account-links'))
+    '&.active::after': {
```

Comment:
Can be simplied by using border-bottom property, and the current page to highlight the link. Also the effect is not consistent across all the links.